augly/video/augmenters/ffmpeg/hflip.py

```diff
@@ -19,4 +30,4 @@ def add_augmenter(
         @returns: a tuple containing the FFMPEG object with the augmentation
             applied and a dictionary with any output arguments as necessary
         """
-        return in_stream.video.hflip(), {}
+        return self.command
```

Comment:
move `command` to be returned & defined in `add_augmenter` (and remove `__init__`)